docs/source/index.md

```diff
@@ -77,6 +77,7 @@ If you have any questions, please feel free to reach out to us on [Twitter](htt
    examples/facesynthetics.ipynb
    examples/synthetic_nlp.ipynb
    examples/multiprocess_dataset_conversion.ipynb
+   examples/spark_dataframe.ipynb
```

Comment:
Can we change the name to `spark_dataframe_to_MDS.ipynb` ?

Reply:
done.